examples/java-agent/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+  id 'java'
+  id 'com.google.cloud.tools.jib' version '0.9.11'
```

Comment:
Oh let's update all these gradle versions to `0.9.12`. Can be a separate PR though.

Reply:
`0.9.12` sounds good.

Reply:
#1139